### ISSUE TITLE
Transparency change

### ISSUE THREAD

**Issue by arafune** [keycastr/keycastr#259]
Hello, 

Thank you for your quick improvement the app on the Japanese keyboard.  I would like to ask you another feature of this app.  

Is it difficult to tune the transparency? 

At present version, the app is not 100% opaque.  I can see the background. This is really nice looking.  However, I sometime feel if more transparent it looks better.  The best transparency may depend on the person to person.  So I think if we can change the transparency in the preference.

Please think about it.

**Comment by arafune** [comment]
I have realized we can change the transparency on the "default" visualizer.  But as far as I know, on Svelte visualizer, we cannot change it.  
So I would like to change my request as follows:
    Can you please improve the app so that we can also change transparency on Svelte?

**Comment by akitchen** [comment]
I agree that the Svelte visualizer could use more options. Originally it had none at all! Some help would be welcome here; this is certainly possible but there are lots of other things to work on first.
